### ISSUE TITLE
Using PyQtWebEngine as option

### ISSUE THREAD

**Issue by mke21** [samuelhwilliams/Eel#423]
In order to be independent of installed browsers (Chromium), is it an idea to add PyQtEngine as an option? It would make a package using Eel fully dependent on pip packages only instead of an installed browser that is not controlled by pip. A developer using Eel only would have to include pyqtwebengine to the requirements in his setup.py file.

In my opinion, the application of Eel doesn't need a full fledged webbrowser anyway, the window needs to display and interprete html. javascript and css well enough to be usefull.

**Comment by ChrisKnott** [comment]
Yes, from a quick test it looks like this will work. I will do some more experiments.

Good idea, thanks.

**Comment by mke21** [comment]
Thanks, I've been looking into it myself already, but I am not familiar enough with the inner workings of Eel to be able to pull it off.

**Comment by SDAMedia** [comment]
is this enhancement already finished ?
